ee/models/event_definition.py

```diff
@@ -10,3 +12,26 @@ class EnterpriseEventDefinition(EventDefinition):
     tags: ArrayField = ArrayField(models.CharField(max_length=32), null=True, blank=True, default=list)
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     updated_by = models.ForeignKey("posthog.User", null=True, on_delete=models.SET_NULL, blank=True)
+    verified: models.BooleanField = models.BooleanField(default=False, null=False, blank=True)
+    verified_at: models.DateTimeField = models.DateTimeField(null=True, blank=True)
+    verified_by = models.ForeignKey(
+        "posthog.User", null=True, on_delete=models.SET_NULL, blank=True, related_name="verifying_user"
+    )
+
+    def __init__(self, *args, **kwargs):
+        # Update verified_at if verified at initialization
+        super().__init__(*args, **kwargs)
+        if self.verified:
+            self.verified_at = timezone.now()
+        self._prev_verified = self.verified
```

Comment:
Not sure if this is the best way to check if the event was previously verified. See suggestion below.